services/wallet/token/tokenstore.go

```diff
@@ -1157,6 +1157,13 @@ func NewDefaultStore() *DefaultStore {
 				Decimals: 6,
 				ChainID:  84532,
 			},
+			&Token{
```

Comment:
@smohamedjavid there is no more this file, please update the `status-go/services/wallet/token/token-lists/default-lists/status.go` with this json object and don't forget to update `Fetched` to the current time.

Reply:
Updated 🙏